storage/range.go

```diff
@@ -956,33 +956,33 @@ func (r *Range) applyRaftCommandInBatch(ctx context.Context, index uint64, origi
 
 // getLeaseForGossip tries to obtain a leader lease. Only one of the replicas
 // should gossip; the bool returned indicates whether it's us.
-func (r *Range) getLeaseForGossip(ctx context.Context) (bool, error) {
+func (r *Range) getLeaseForGossip(ctx context.Context) (hasLease bool, err error) {
```

Comment:
Did you only introduce the named return to explain the parameter? I prefer only using them when there's a good reason.